_data/partner.yml

```
@@ -814,4 +814,22 @@
   logo: image/partners/COSCon2018/sxl.png
   records:
     - year: 2018
-      type: 网站支持
\ No newline at end of file
+      type: 网站支持
+- title: 欧侃在线
+  site: www.oukan.online
```

```suggestion
  site: http://www.linuxren.org.cn/
```
1. **网址协议前缀**确实会让网页上变成错误的相对路径
2. 该域名访问时实际会重定向到 http://www.linuxren.org.cn/